nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -321,5 +321,5 @@ nifi.analytics.connection.model.score.name=${nifi.analytics.connection.model.sco
 nifi.analytics.connection.model.score.threshold=${nifi.analytics.connection.model.score.threshold}
 
 # runtime monitoring properties
-nifi.monitor.long.running.task.schedule=${nifi.monitor.long.running.task.schedule}
-nifi.monitor.long.running.task.threshold=${nifi.monitor.long.running.task.threshold}
+nifi.monitor.long.running.task.schedule=
```

Comment:
I think it's a good practice to include the measurement unit in the property name or to mention it in a comment above. Relying on other, already provided property values, could be either 10 secs or 10 sec, or just 10.

Reply:
There is no single measurement unit that must be used but the property can be configured like other time period type properties: 60 sec, 60 secs, 60 seconds, 1 min, etc.
It is mentioned in the Admin Guide that these are time period values with an example value.